scripts/enrich-proposal.sh

```
@@ -0,0 +1,219 @@
+# Each command handles its own errors via || true or conditional checks.
+
+PROJECT_DIR="${1:-.}"
+PROPOSAL="${2:-}"
+
+if [ -z "$PROPOSAL" ] || [ ! -f "$PROPOSAL" ]; then
+  echo "Usage: enrich-proposal.sh <project-dir> <proposal-path>"
+  echo "Error: Proposal file not found: $PROPOSAL"
+  exit 1
+fi
+
+cd "$PROJECT_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)